Assets/Developers/Lab/Components/DayNight.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Unity.Collections;
+using Unity.Entities;
+using Unity.Mathematics;
+using UnityEngine;
+
+
+namespace JanusVagabond
+{
+  public enum DayNightPhase
+  {
+    DAY = 0,
+    NIGHT = 1
+  }
+
+  namespace Data {
+    [Serializable]
+    public struct DayNight : IComponentData
+    {
+      public DayNightPhase Phase;
```

Comment:
Not using E for Enum naming convention?

Reply:
That's the UE4 C++ convention. For C# reference MSDN: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members